.github/workflows/benchmark.yml

```diff
@@ -103,17 +103,29 @@ jobs:
     - name: Install deps
       run: sudo apt -y install protobuf-compiler
 
+    # TODO: Should be removed once frame-omni-bencher can be installed using
+    # toolchain defined in the `rust-toolchain.toml` file.
+    - name: Install latest stable Rust
+      run: rustup install stable
+
+    - name: Set stable as default
+      run: rustup default stable
+
+    - name: Install targets and components
+      run: |
+        rustup target add wasm32-unknown-unknown
+        rustup component add rust-src
+
+    # TODO: Should be moved below the `rustup show` once the toolchain
+    - name: Install frame-omni bencher tool
+      run: cargo install --git https://github.com/paritytech/polkadot-sdk frame-omni-bencher --profile=production --locked
```

Comment:
can't we do `cargo +nightly`?

Reply:
I can specify the toolchain, but I still have to install it.

Don't know what's avaialble on our custom machine in advance (and it can change).